cmd/ausoceantv/auth.go

```diff
@@ -0,0 +1,63 @@
+/*
+DESCRIPTION
+
```

Comment:
These extra lines after the titles aren't consistent with other comment headers and should be removed.